src/Lachain.Console/Application.cs

```diff
@@ -92,15 +99,33 @@ public void Start(RunOptions options)
             commandManager.Start(wallet.EcdsaKeyPair);
             rpcManager.Start();
 
+            consensusManager.Start((long)blockManager.GetHeight() + 1);
+            validatorStatusManager.Start(false);
+
+            Thread.Sleep(10000);
+
+            if (blockManager.GetHeight() == 0)
+            {
+                blockSynchronizer.StartFastSync();    
+            }
+            
+            // consensusManager.Start((long)blockManager.GetHeight() + 1);
+            // validatorStatusManager.Start(false);
+
```

Comment:
```suggestion
            if (blockManager.GetHeight() == 0)
            {
                blockSynchronizer.StartFastSync();    
            }
            
            consensusManager.Start((long)blockManager.GetHeight() + 1);
            validatorStatusManager.Start(false);
```
This variant is working for me in local testnet,  in other case consensusManager causes errors during the sync

Reply:
This will not be the final state, this I've used so that on local I can test the FastSync